applications/PFEMapplication/CMakeLists.txt~

```diff
@@ -0,0 +1,46 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
you should add ~ files to your gitignore. This file should not be added to repository